compiler/ballerina-parser/src/test/java/io/ballerinalang/compiler/parser/test/syntax/types/ArrayTypeTest.java

```diff
@@ -104,6 +104,11 @@ public void testInvalidArrayTypeWithArrayLengthMissingCloseBracket() {
         test("int[5 a;", "array-type/array_type_assert_11.json");
     }
 
+    @Test
+    public void testTopLevelArrayTypeWithMissingCloseBracket() {
+        testTopLevelNode("array-type/array_type_assert_05.bal", "array-type/array_type_assert_20.json");
```

Comment:
```suggestion
        testTopLevelNode("array-type/array_type_assert_20.bal", "array-type/array_type_assert_20.json");
      
```
Can we sync bal file and assert file. It would be better if we can change the existing ones as well.

Reply:
array_type_**source**_20.bal :P